src/Annotator.js

```diff
@@ -970,11 +989,11 @@ class Annotator extends EventEmitter {
                 break;
```

Comment:
Should the enclosing function be instead called `handleServiceErrors()`? It's weird that this says it handles all events, and then only ends up handling error messaging

Reply:
Agreed. Renamed it